charts/k8ssandra/templates/reaper/reaper.yaml

```diff
@@ -32,14 +32,14 @@ spec:
   {{- if .Values.reaper.tolerations }}
   tolerations: {{ toYaml .Values.reaper.tolerations | nindent 4 }}
   {{- end }}
+  {{- if (and .Values.reaper.schemaInitContainerConfig .Values.reaper.schemaInitContainerConfig.securityContext) }}
```

Comment:
In this change, if the context in yaml is specified as { } , which is our default, then we will apply whatever else is in the schema and config blocks. 

What if the user has a value setting for config containers without a securityContext entry? It would be null and we would disregard any ContainerConfig contents.  Wondering if it would be better to default the configs irrespective of the securityContext or maybe I'm missing something here.

Reply:
The security context is the only thing that we are exposing for those init containers so the checks should be fine as is.